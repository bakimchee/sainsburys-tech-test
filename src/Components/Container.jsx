import styled from 'styled-components';
import { theme } from '../Theme';

const Container = styled.div`
  display: block;
  padding: 0 ${theme.size.spacing.large};
  max-width: 960px;
  margin: 0 auto;
`;

export default Container;
