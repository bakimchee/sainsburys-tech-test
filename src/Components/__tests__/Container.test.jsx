/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import Container from '../Container';

test('Container', () => {
  const component = renderer.create(<Container>Content</Container>);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
