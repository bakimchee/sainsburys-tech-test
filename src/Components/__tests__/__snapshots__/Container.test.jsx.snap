// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container 1`] = `
<div
  className="Container-u58elu-0 ixyjYR"
>
  Add to basket
</div>
`;
